bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -133,6 +133,31 @@ <h3>Tracking Protection</h3>
     <li>shavar.services.mozilla.com</li>
   </ul>
 
+  <h2 id="pocket"><b>Pocket</b></h2>
+
+  <p>Pocket pays out bounties at a reduced rate and only if the reported bug has Critical or High severity rating. The scope for Pocket in this program is limited to the below targets. Any API endpoint or path not explicitly mentioned is considered out of scope.</p>
+
+  <ul class="mzp-u-list-styled">
+    <li>Pocket Web application under the following paths:
+      <ul>
+        <li>getpocket.com/home</li>
+        <li>getpocket.com/account</li>
+        <li>getpocket.com/discover</li>
+        <li>getpocket.com/collections</li>
+        <li>getpocket.com/my-list/*</li>
+        <li>getpocket.com/read/*</li>
+      </ul>
+    </li>
+    <li>API endpoints listed in https://getpocket.com/developer/ which include:
```

Comment:
I'm assuming this means that things like the [Preferences API](https://getpocket.com/developer/docs/v3/preferences-api) and [Article View API](https://getpocket.com/developer/docs/v3/article-view) are out of scope.

If so we should probably change this to

> The following API endpoints described in https://getpocket.com/developer/

because otherwise it's ambiguous if it means those 3 endpoints, or the endpoints listed on https://getpocket.com/developer/

Reply:
maybe it's clearer to say "the following API endpoints" without referencing the developer page, https://getpocket.com/developer/

Reply:
Ah yea, I meant exactly what @fkiriakos07 put in the documentation here :)